src/ui/public/paginated_table/paginated_table.html

```diff
@@ -28,7 +28,7 @@
       <tbody kbn-rows="page" kbn-rows-min="paginatedTable.rowsToShow(perPage, page.length)"></tbody>
       <tfoot ng-if="showTotal">
         <tr>
-          <th ng-repeat="col in columns" class="numeric-value">{{col.total | number}}</th>
+          <th ng-repeat="col in columns" class="numeric-value">{{col.total}}</th>
```

Comment:
@ppisljar, I'm not sure I understand you.  I can't use the fieldformatter from the field if the 'count' function is requested.  Otherwise, a date field will format the 'count' value as a date which won't make any sense.